aci/aci-socat/runlevels/builder/10.builder.sh

```diff
@@ -24,7 +24,7 @@ CC='/usr/bin/gcc -static' CFLAGS='-fPIC' ./configure \
 	--enable-static
 
 cd /build
-curl -LO ftp://ftp.cwru.edu/pub/bash/readline-${READLINE_VERSION}.tar.gz
+curl -LO https://ftp.gnu.org/gnu/readline/readline-${READLINE_VERSION}.tar.gz
```

Comment:
Would it be easier to trash READLINE_VERSION for the place to where we download 
w/h -o readline.tar
and --transform in tar
cd realine
Seems less of a pressure to know READLINE_VERSION
But it might be to much